File: withFunctions/fromFahrenheitToCelcius.java
package springIntro;

import java.util.*;

public class fromFahrenheitToCelcius {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		System.out.printf("Fahrenheit value = ");
		float fahn = in.nextFloat();
		float celc = trans(fahn);
		System.out.println("Celcius Value = " + celc);
	}
	static float trans(float fahn) {
		return (float) ((fahn - 32) / 1.8);
	}

}

File: withFunctions/fromFahrenheitToCelcius.cpp
#include<iostream>
using namespace std;
void trans(float fahn, float celc){
	celc = (fahn - 32) / 1.8;
	printf("Celcius value = %.2f", celc);
}
int main(){
	int fahn, celc;
	cout << "Enter a fahrenheit value = ";
	cin >> fahn;
	trans(fahn, celc);
	return 0;
}
